### PR TITLE
Simplify minikube getting started guide

### PR DESCRIPTION
**Pull request by tgraf**
Depends on #6495

---
This change is [Reviewable](https://reviewable.io/reviews/cilium/cilium/6496)

### CODE REVIEW

**Comment by tgraf** [comment]
test-me-please

**Comment by tgraf** [comment]
test-me-please

**Comment by tgraf** [comment]
test-me-please

File: Documentation/gettingstarted/minikube.rst [hunk 1]
```diff
@@ -20,6 +20,441 @@ deploying Cilium on a full fledged Kubernetes cluster, then go straight to
...
+    po/deathstar-76995f4687-xbgnl    1/1       Running   0          1m
+    po/tiefighter                    1/1       Running   0          1m
+    po/xwing                         1/1       Running   0          1m
+
+    NAME             TYPE        CLUSTER-IP       EXTERNAL-IP   PORT(S)   AGE
+    svc/deathstar    ClusterIP   10.109.254.198   <none>        80/TCP    3h
+    svc/kubernetes   ClusterIP   10.96.0.1        <none>        443/TCP   3h
+
+Each pod will be represented in Cilium as an :ref:`endpoint`. We can invoke the
+``cilium`` tool inside the Cilium pod to list them:
```
Comment (eloycoto) [inline Documentation/gettingstarted/minikube.rst:128]: should the `kubectl get cep` be used here?

**Review by eloycoto — APPROVED** [review]
Guide LGTM, the only concern is we should use `kubectl get cep`

File: Documentation/gettingstarted/minikube.rst (hunk 1 above)
Comment (tgraf) [reply Documentation/gettingstarted/minikube.rst:128]: This is from the existing guide. I've only reworked the Cilium installation part for now.

**Pull request merged**